cves/CVE-2012-0022.yml

```diff
@@ -206,4 +218,11 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: From what I have seen, it seems like a design mistake led to this
+    vulnerability. It appears that when created, the original code did not
+    account for the possibly of large/excessive parameters.This could be due to
+    the fact that the developer never anticipated that their system could be
+    abused in such a way. I did find it interesting that the vulnerability was
```

Comment:
Instead of "I did find it interesting", should probably be "I found it interesting" or "I also found it interesting". (Passive voice)

Reply:
Fixed this in latest commit.